js-exercises/animal-sort/animalSort.js

```diff
@@ -0,0 +1,24 @@
+const animalSort = animals => {
+  if (!(animals instanceof Array)) {
+    throw Error(`Expected Array as an input, got ${typeof animals}`);
+  }
+
+  if (animals.length === 0) {
+    return [];
+  }
+
+  return animals.sort((animalA, animalB) => {
+    if (animalA.numberOfLegs > animalB.numberOfLegs) {
+      return 1;
+    }
+    if (animalA.numberOfLegs === animalB.numberOfLegs) {
+      return animalA.name.localeCompare(animalB.name);
+    }
+    if (animalA.numberOfLegs < animalB.numberOfLegs) {
+      return -1;
+    }
+    return 1;
```

Comment:
```suggestion
   if (animalA.numberOfLegs === animalB.numberOfLegs) {
      return animalA.name.localeCompare(animalB.name);
    }
   return animalA.numberOfLegs > animalB.numberOfLegs;
```

Reply:
Thanks @varora1406 for that

Reply:
That doesn't work the same. Can you check @varora1406 ?